src/main/java/duke/Command.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.util.List;
+
+public class Command {
+    private final String description;
+    private final List<String> arguments;
+
+    /**
+     * Constructs a Command with the specified description and any necessary arguments.
+     * 
+     * @param description The Command description (eg: todo, list, mark, etc)
+     * @param arguments The necessary arguments for the command (eg: command "deadline" can have argument "11/02/2023 1200"
```

Comment:
Would like to check if it is possible for comments to be wrapped. This line is getting long